drivers/ukbus/pci/arch/arm64/pci_bus.c

```diff
@@ -95,14 +95,16 @@ static int arch_pci_driver_add_device(struct pci_driver *drv,
 	dev->irq = irq;
 	uk_pr_info("pci dev base(0x%lx) irq(%ld)\n", dev->base, dev->irq);
 
-	if (drv->add_dev)
+	if (drv->add_dev) {
 		ret = drv->add_dev(dev); //virtio pci
-	if (ret < 0) {
-		uk_pr_err("PCI %02x:%02x.%02x: Failed to initialize device driver\n",
-			  (int) addr->bus,
-			  (int) addr->devid,
-			  (int) addr->function);
-		uk_free(pha, dev);
+		if (unlikely(ret < 0)) {
+			uk_pr_err("PCI %02x:%02x.%02x: Failed to initialize device driver\n",
+				  (int) addr->bus,
```

Comment:
I know this was present here before, but the only other `checkpatch.pl --strict` `CHECK` I am getting in this PR is regarding the whitespace after the cast. I know this is going to be deleted anyway when it will be reworked so probably no point in pointing this out now, but I thought I'd let you know in case that maybe you missed this and you would have wanted to modify it regardless. I leave it up to you if you want to waste some more time modifying this or not.